CMakeLists.txt

```diff
@@ -131,7 +129,6 @@ if (NOT WIN32)
     endif()
```

Comment:
I believe we will have to remove the above testing section as well. No use to lug around the `kmstool-enclaves` tests if we remove `kmstool-enclave`.

I am not completely sure though. Maybe that test makes sense even without the integration script?